### PR TITLE
Metrics Server: print versions during startup

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

Minor improvement for Metric Server -  let's print version info during startup, the same way KEDA Operator is doing currently

### CODE REVIEW

**Comment by tomkerkhove** [comment]
Will leave it up to @ahmelsayed to review code but LGTM - Like the idea!

File: Makefile [hunk 1]
```diff
@@ -98,12 +98,12 @@ build: checkenv build-adapter build-controller
 .PHONY: build-controller
 build-controller: generate-api pkg/scalers/liiklus/LiiklusService.pb.go
 	$(GO_BUILD_VARS) operator-sdk build $(IMAGE_CONTROLLER) \
-		--go-build-args "-ldflags -X=main.GitCommit=$(GIT_COMMIT) -ldflags -X=github.com/kedacore/keda/version.Version=$(VERSION) -o build/_output/bin/keda"
+		--go-build-args "-ldflags -X=github.com/kedacore/keda/version.GitCommit=$(GIT_COMMIT) -ldflags -X=github.com/kedacore/keda/version.Version=$(VERSION) -o build/_output/bin/keda"
```
Comment (ahmelsayed) [inline Makefile:101]: I think this doesn't actually work, right? The old one didn't, `GitCommit` was always empty. I looked up the issue and had a change for it, but I never pushed it. `operator-sdk` doesn't support multiple AFAIK https://github.com/operator-framework/operator-sdk/issues/1867. You can just remove the second `-ldflags` arg

Comment (zroubalik) [reply Makefile:101]: Haven't noticed that, thanks!

Comment (zroubalik) [reply Makefile]: Fixed

**Comment by zroubalik** [comment]
Seems like, operator-sdk v0.12 can't handle multiple -ldflag fields. I have locally newer version (the one present in v2) which is working correctly.
I think we can disable printin GitCommit for now and enable it in the v2 branch separately.

**Pull request merged**